classification2.Rmd

```diff
@@ -52,23 +52,28 @@ the observations in the test set? One way we can do this is to calculate the
 **prediction accuracy**. This is the fraction of examples for which the
 classifier made the correct prediction. To calculate this we divide the number
 of correct predictions by the number of predictions made. 
-This process is illustrated in Figure \@ref(fig:06-ML-paradigm-test).
-Note that there are other measures for how well classifiers perform, such as *precision* and *recall*; 
+
+$$prediction \; accuracy = \frac{number \; of  \; correct  \; predictions}{total \;  number \;  of  \; predictions}$$
+
+
+The process for assessing if our predictions match the true labels in the 
+test set is illustrated in Figure \@ref(fig:06-ML-paradigm-test). Note that there 
+are other measures for how well classifiers perform, such as *precision* and *recall*; 
 these will not be discussed here, but you will likely encounter them in other more advanced
 books on this topic. 
 
 ```{r 06-ML-paradigm-test, echo = FALSE, message = FALSE, warning = FALSE, fig.cap = "Process for splitting the data and finding the prediction accuracy", fig.retina = 2, out.width = "800"}
 knitr::include_graphics("img/ML-paradigm-test.png")
 ```
 
-In R, we can use the `tidymodels` library collection not only to perform K-nearest neighbours
+In R, we can use the `tidymodels` package not only to perform $K$-nearest neighbors
 classification, but also to assess how well our classification worked. Let's
-work through an example of this process using the breast cancer dataset.
-We start by loading the necessary libraries, reading in the breast cancer data
+work through an example of this process using the breast cancer data set.
+We start by loading the necessary packages, reading in the breast cancer data
 from the previous chapter, and making a quick scatter plot visualization of
-tumour cell concavity versus smoothness coloured by diagnosis in Figure \@ref(fig:06-precode).
+tumor cell concavity versus smoothness colored by diagnosis in Figure \@ref(fig:06-precode).
 
-```{r 06-precode, fig.height = 4, fig.width = 5, fig.cap="Scatterplot of tumour cell concavity versus smoothness coloured by diagnosis label", message = F, warning = F}
+```{r 06-precode, fig.height = 4, fig.width = 5, fig.cap="Scatterplot of tumor cell concavity versus smoothness coloured by diagnosis label", message = F, warning = F}
```

Comment:
scatterplot -> scatter plot 

(I believe that is what is more consistently used throughout)

Reply:
also just in general i prefer scatter plot